src/chainer.js

```diff
@@ -8,11 +10,12 @@ function chainer(functions) {
     let result = x;
 
     for (const f of functions) {
-      result = f(result);
+      if (typeof f === 'function') {
+        result = f(result);
+      }
     }
-
     return result;
-  }
+  };
 }
```

Comment:
The function chainer is not properly closed with a correct syntax. This is likely to cause a syntax error. The correct way should be 'function chainer(functions) {...}' without the extra closing bracket '}'.